src/test/java/test/integration/extension/AddPackages.java

```diff
@@ -1,19 +1,25 @@
 package test.integration.extension;
 
 import java.lang.annotation.ElementType;
+import java.lang.annotation.Repeatable;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
 @Target(ElementType.TYPE)
 @Retention(RetentionPolicy.RUNTIME)
+@Repeatable(AddPackagesGroup.class)
 public @interface AddPackages {
```

Comment:
It still allows you to add more than one package as `value` or `containing`... So either we limit that choice and make `@AddPackage` redundant... or ... ?